pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -354,7 +353,7 @@ func genCredentials() error {
 	err := readCredentials()
 	if err != nil {
 		// Generate a new uuid
-		out, err := exec.Command("uuidgen").Output()
+		out, err := base.ExecCommand(log, false, false, "uuidgen")
```

Comment:
Wait, this should use the uuid package to generate a uuid; no need for an Exec.

Reply:
Done.